families.py

```diff
@@ -29,6 +29,7 @@ def jacobi_recurrence_values(N, alpha, beta):
                       sp.gammaln(alpha + 1.) + sp.gammaln(beta + 1.) -
                       sp.gammaln(alpha + beta + 2.)
                     )
+        ab = np.sqrt(ab)
```

Comment:
Why this change? Was the `sqrt` not there before? This change doesn't seem relevant to the exponential/normal distributions.